docs/AUGUR_INT.md

```diff
@@ -0,0 +1,64 @@
+# **8Knot+Augur Integration**
+
+## **Intro**
+---
+
+A user should be able to:
+1. Create a group of repos and orgs that they want to see aggregate for and
+2. Access that group in 8Knot.
+
+## **Preparation**
+---
+1. Clone [github.com/oss-aspen/8Knot/augur_int]
+   - We use a docker compose multi-container application build strategy.
+   - Ports 8050 (*Dash*) and 6379 (*Redis*) are assumed to be available. If this isn't possible, modifications to the docker-compose.yml file may be necessary.
+   - Inter-pod name resolution is done with Docker network's DNS.
+
+2. Create an application in Augur front-end
+   - Create an account at chaoss.tv:5038.
```

Comment:
need to change to the generic term for chaoss.tv

Reply:
++